2020-07-04 11:08:50,984 - engineio.server - INFO - Server initialized for eventlet.
2020-07-04 11:08:50,999 - werkzeug - INFO -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2020-07-04 11:08:55,771 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:08:55] "[37mPOST /organizations HTTP/1.1[0m" 200 -
2020-07-04 11:08:55,779 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3f14be0>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************st"]', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:08:55,780 - meraki - DEBUG - {'tags': ['Organizations'], 'operation': 'getOrganizations', 'method': 'GET', 'url': '/organizations', 'params': None}
2020-07-04 11:08:55,784 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:08:55,944 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations HTTP/1.1" 404 None
2020-07-04 11:08:55,946 - meraki - ERROR - Organizations, getOrganizations - 404 Not Found, b' '
2020-07-04 11:08:56,230 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:08:56] "[37mGET /organizations HTTP/1.1[0m" 200 -
2020-07-04 11:09:00,097 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:00] "[37mPOST /organizations HTTP/1.1[0m" 200 -
2020-07-04 11:09:00,102 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3668820>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:00,103 - meraki - DEBUG - {'tags': ['Organizations'], 'operation': 'getOrganizations', 'method': 'GET', 'url': '/organizations', 'params': None}
2020-07-04 11:09:00,105 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:00,250 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations HTTP/1.1" 302 None
2020-07-04 11:09:00,255 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:01,236 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations HTTP/1.1" 200 None
2020-07-04 11:09:01,237 - meraki - INFO - Organizations, getOrganizations - 200 OK
2020-07-04 11:09:01,238 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:01] "[37mGET /organizations HTTP/1.1[0m" 200 -
2020-07-04 11:09:02,357 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:02] "[37mPOST /networks HTTP/1.1[0m" 200 -
2020-07-04 11:09:02,360 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a36820a0>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:02,362 - meraki - DEBUG - {'tags': ['Networks'], 'operation': 'getOrganizationNetworks', 'method': 'GET', 'url': '/organizations/764320/networks', 'params': {}}
2020-07-04 11:09:02,364 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:02,489 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/networks HTTP/1.1" 302 None
2020-07-04 11:09:02,492 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:02,812 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/networks HTTP/1.1" 200 None
2020-07-04 11:09:02,814 - meraki - INFO - Networks, getOrganizationNetworks - 200 OK
2020-07-04 11:09:02,815 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:02] "[37mGET /networks HTTP/1.1[0m" 200 -
2020-07-04 11:09:04,566 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:04] "[37mPOST /clients HTTP/1.1[0m" 200 -
2020-07-04 11:09:04,570 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a368e820>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:04,570 - meraki - DEBUG - {'tags': ['Clients'], 'operation': 'getNetworkClients', 'page': 1}
2020-07-04 11:09:04,571 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:04,696 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/clients?perPage=1000&timespan=3600 HTTP/1.1" 302 None
2020-07-04 11:09:04,701 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:05,082 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/clients?perPage=1000&timespan=3600&perPage=1000&timespan=3600 HTTP/1.1" 200 None
2020-07-04 11:09:05,085 - meraki - INFO - Clients, getNetworkClients; page 1 - 200 OK
2020-07-04 11:09:05,087 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:05] "[37mGET /clients HTTP/1.1[0m" 200 -
2020-07-04 11:09:10,486 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:10] "[37mPOST /devices HTTP/1.1[0m" 200 -
2020-07-04 11:09:10,493 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a368ee50>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:10,496 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:10] "[37mPOST /device_status HTTP/1.1[0m" 200 -
2020-07-04 11:09:10,497 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3693a90>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:10,504 - meraki - DEBUG - {'tags': ['Organizations'], 'operation': 'getOrganizationDeviceStatuses', 'method': 'GET', 'url': '/organizations/764320/deviceStatuses', 'params': None}
2020-07-04 11:09:10,501 - meraki - DEBUG - {'tags': ['Devices'], 'operation': 'getNetworkDevices', 'method': 'GET', 'url': '/networks/L_681169443639788368/devices', 'params': None}
2020-07-04 11:09:10,502 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3630dc0>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:10,499 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:10] "[37mPOST /uplink_loss HTTP/1.1[0m" 200 -
2020-07-04 11:09:10,509 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:10,509 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:10,512 - meraki - DEBUG - {'tags': ['Organizations'], 'operation': 'getOrganizationUplinksLossAndLatency', 'method': 'GET', 'url': '/organizations/764320/uplinksLossAndLatency', 'params': {}}
2020-07-04 11:09:10,518 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:10,644 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/deviceStatuses HTTP/1.1" 302 None
2020-07-04 11:09:10,649 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:10,654 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/uplinksLossAndLatency HTTP/1.1" 302 None
2020-07-04 11:09:10,657 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:10,659 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/devices HTTP/1.1" 302 None
2020-07-04 11:09:10,662 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:10,929 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/uplinksLossAndLatency HTTP/1.1" 200 None
2020-07-04 11:09:10,930 - meraki - INFO - Organizations, getOrganizationUplinksLossAndLatency - 200 OK
2020-07-04 11:09:10,931 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:10] "[37mGET /uplink_loss HTTP/1.1[0m" 200 -
2020-07-04 11:09:11,020 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/deviceStatuses HTTP/1.1" 200 None
2020-07-04 11:09:11,024 - meraki - INFO - Organizations, getOrganizationDeviceStatuses - 200 OK
2020-07-04 11:09:11,026 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:11] "[37mGET /device_status HTTP/1.1[0m" 200 -
2020-07-04 11:09:11,265 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/devices HTTP/1.1" 200 None
2020-07-04 11:09:11,267 - meraki - INFO - Devices, getNetworkDevices - 200 OK
2020-07-04 11:09:11,268 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:11] "[37mGET /devices HTTP/1.1[0m" 200 -
2020-07-04 11:09:31,156 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:31,333 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/admins HTTP/1.1" 302 None
2020-07-04 11:09:31,337 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:31,720 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/admins HTTP/1.1" 200 None
2020-07-04 11:09:31,722 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a369c040>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:31,722 - meraki - DEBUG - {'tags': ['MX VPN firewall'], 'operation': 'getOrganizationVpnFirewallRules', 'method': 'GET', 'url': '/organizations/764320/vpnFirewallRules', 'params': None}
2020-07-04 11:09:31,724 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:31,911 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/vpnFirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:31,915 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:32,223 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/vpnFirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:32,224 - meraki - INFO - MX VPN firewall, getOrganizationVpnFirewallRules - 200 OK
2020-07-04 11:09:32,277 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/snmp HTTP/1.1" 302 None
2020-07-04 11:09:32,509 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/snmp HTTP/1.1" 200 None
2020-07-04 11:09:32,614 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/thirdPartyVPNPeers HTTP/1.1" 302 None
2020-07-04 11:09:32,881 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/thirdPartyVPNPeers HTTP/1.1" 200 None
2020-07-04 11:09:32,934 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368 HTTP/1.1" 302 None
2020-07-04 11:09:33,179 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368 HTTP/1.1" 200 None
2020-07-04 11:09:33,235 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/vlansEnabledState HTTP/1.1" 302 None
2020-07-04 11:09:33,481 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/vlansEnabledState HTTP/1.1" 200 None
2020-07-04 11:09:33,537 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/vlans HTTP/1.1" 302 None
2020-07-04 11:09:33,878 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/vlans HTTP/1.1" 200 None
2020-07-04 11:09:34,387 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/cellularFirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:34,971 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/cellularFirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:35,089 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:35,351 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:35,472 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/siteToSiteVpn HTTP/1.1" 302 None
2020-07-04 11:09:35,704 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/siteToSiteVpn HTTP/1.1" 200 None
2020-07-04 11:09:35,765 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids HTTP/1.1" 302 None
2020-07-04 11:09:36,053 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids HTTP/1.1" 200 None
2020-07-04 11:09:36,107 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/0/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:36,370 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/0/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:36,419 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/1/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:36,760 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/1/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:36,809 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/2/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:37,088 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/2/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:37,143 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/3/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:37,557 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/3/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:37,607 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/4/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:37,851 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/4/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:37,899 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/5/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:38,136 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/5/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:38,187 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/6/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:38,465 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/6/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:38,570 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/7/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:38,977 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/7/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:39,030 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/8/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:39,277 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/8/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:39,334 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/9/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:39,562 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/9/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:39,616 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/10/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:39,887 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/10/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:39,940 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/11/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:40,308 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/11/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:40,360 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/12/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:40,730 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/12/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:40,785 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/13/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:41,069 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/13/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:41,127 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/14/l3FirewallRules HTTP/1.1" 302 None
2020-07-04 11:09:41,344 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/ssids/14/l3FirewallRules HTTP/1.1" 200 None
2020-07-04 11:09:41,418 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/devices HTTP/1.1" 302 None
2020-07-04 11:09:41,911 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/devices HTTP/1.1" 200 None
2020-07-04 11:09:41,913 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3638310>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:09:41,914 - meraki - DEBUG - {'tags': ['Devices'], 'operation': 'getNetworkDevices', 'method': 'GET', 'url': '/networks/L_681169443639788368/devices', 'params': None}
2020-07-04 11:09:41,915 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:09:42,066 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/devices HTTP/1.1" 302 None
2020-07-04 11:09:42,070 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:09:42,539 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/devices HTTP/1.1" 200 None
2020-07-04 11:09:42,544 - meraki - INFO - Devices, getNetworkDevices - 200 OK
2020-07-04 11:09:42,544 - meraki - DEBUG - {'tags': ['Switch ports'], 'operation': 'getDeviceSwitchPorts', 'method': 'GET', 'url': '/devices/Q2HP-QSVT-LBLV/switchPorts', 'params': None}
2020-07-04 11:09:42,824 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/devices/Q2HP-QSVT-LBLV/switchPorts HTTP/1.1" 200 None
2020-07-04 11:09:42,825 - meraki - INFO - Switch ports, getDeviceSwitchPorts - 200 OK
2020-07-04 11:09:42,826 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:09:42] "[37mPOST /run_backup/ HTTP/1.1[0m" 200 -
2020-07-04 11:10:06,026 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3638d90>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:10:06,032 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:06] "[37mPOST /device_status HTTP/1.1[0m" 200 -
2020-07-04 11:10:06,034 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3630bb0>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:10:06,038 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:06] "[37mPOST /devices HTTP/1.1[0m" 200 -
2020-07-04 11:10:06,044 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3640040>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:10:06,047 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:06] "[35m[1mGET /devices HTTP/1.1[0m" 500 -
2020-07-04 11:10:06,049 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:06] "[37mPOST /uplink_loss HTTP/1.1[0m" 200 -
2020-07-04 11:10:06,065 - meraki - DEBUG - {'tags': ['Organizations'], 'operation': 'getOrganizationUplinksLossAndLatency', 'method': 'GET', 'url': '/organizations/764320/uplinksLossAndLatency', 'params': {}}
2020-07-04 11:10:06,065 - meraki - DEBUG - {'tags': ['Organizations'], 'operation': 'getOrganizationDeviceStatuses', 'method': 'GET', 'url': '/organizations/764320/deviceStatuses', 'params': None}
2020-07-04 11:10:06,081 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:10:06,088 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:10:06,093 - werkzeug - ERROR - Error on request:
Traceback (most recent call last):
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_socketio/__init__.py", line 45, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/engineio/middleware.py", line 74, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cyberdevnet/mer-hacker-dev/api/api.py", line 118, in get_devices
    data['networkId'])
KeyError: 'networkId'
2020-07-04 11:10:06,257 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/deviceStatuses HTTP/1.1" 302 None
2020-07-04 11:10:06,261 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:10:06,272 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/uplinksLossAndLatency HTTP/1.1" 302 None
2020-07-04 11:10:06,277 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:10:06,673 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/deviceStatuses HTTP/1.1" 200 None
2020-07-04 11:10:06,673 - meraki - INFO - Organizations, getOrganizationDeviceStatuses - 200 OK
2020-07-04 11:10:06,674 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:06] "[37mGET /device_status HTTP/1.1[0m" 200 -
2020-07-04 11:10:06,935 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/uplinksLossAndLatency HTTP/1.1" 200 None
2020-07-04 11:10:06,936 - meraki - INFO - Organizations, getOrganizationUplinksLossAndLatency - 200 OK
2020-07-04 11:10:06,938 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:06] "[37mGET /uplink_loss HTTP/1.1[0m" 200 -
2020-07-04 11:10:19,144 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:19] "[37mPOST /clients HTTP/1.1[0m" 200 -
2020-07-04 11:10:19,148 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:19] "[37mPOST /networks HTTP/1.1[0m" 200 -
2020-07-04 11:10:19,153 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:19] "[35m[1mGET /networks HTTP/1.1[0m" 500 -
2020-07-04 11:10:19,165 - werkzeug - ERROR - Error on request:
Traceback (most recent call last):
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_socketio/__init__.py", line 45, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/engineio/middleware.py", line 74, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cyberdevnet/mer-hacker-dev/api/api.py", line 94, in get_networks
    dashboard = meraki.DashboardAPI(
  File "/home/cyberdevnet/mer-hacker-dev/api/meraki/__init__.py", line 130, in __init__
    raise APIKeyError()
meraki.exceptions.APIKeyError: Meraki API key needs to be defined
2020-07-04 11:10:19,170 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:19] "[35m[1mGET /clients HTTP/1.1[0m" 500 -
2020-07-04 11:10:19,178 - werkzeug - ERROR - Error on request:
Traceback (most recent call last):
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_socketio/__init__.py", line 45, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/engineio/middleware.py", line 74, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cyberdevnet/mer-hacker-dev/api/api.py", line 159, in clients
    dashboard = meraki.DashboardAPI(ARG_APIKEY)
  File "/home/cyberdevnet/mer-hacker-dev/api/meraki/__init__.py", line 130, in __init__
    raise APIKeyError()
meraki.exceptions.APIKeyError: Meraki API key needs to be defined
2020-07-04 11:10:27,065 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:27] "[37mPOST /organizations HTTP/1.1[0m" 200 -
2020-07-04 11:10:27,074 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:27] "[35m[1mGET /organizations HTTP/1.1[0m" 500 -
2020-07-04 11:10:27,083 - werkzeug - ERROR - Error on request:
Traceback (most recent call last):
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_socketio/__init__.py", line 45, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/engineio/middleware.py", line 74, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cyberdevnet/mer-hacker-dev/api/api.py", line 72, in get_organizations
    dashboard = meraki.DashboardAPI(data['X-Cisco-Meraki-API-Key'],
  File "/home/cyberdevnet/mer-hacker-dev/api/meraki/__init__.py", line 130, in __init__
    raise APIKeyError()
meraki.exceptions.APIKeyError: Meraki API key needs to be defined
2020-07-04 11:10:29,709 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:29] "[37mPOST /organizations HTTP/1.1[0m" 200 -
2020-07-04 11:10:29,715 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:29] "[35m[1mGET /organizations HTTP/1.1[0m" 500 -
2020-07-04 11:10:29,724 - werkzeug - ERROR - Error on request:
Traceback (most recent call last):
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_socketio/__init__.py", line 45, in __call__
    return super(_SocketIOMiddleware, self).__call__(environ,
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/engineio/middleware.py", line 74, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cyberdevnet/.local/lib/python3.8/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cyberdevnet/mer-hacker-dev/api/api.py", line 72, in get_organizations
    dashboard = meraki.DashboardAPI(data['X-Cisco-Meraki-API-Key'],
  File "/home/cyberdevnet/mer-hacker-dev/api/meraki/__init__.py", line 130, in __init__
    raise APIKeyError()
meraki.exceptions.APIKeyError: Meraki API key needs to be defined
2020-07-04 11:10:30,464 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:30] "[37mPOST /networks HTTP/1.1[0m" 200 -
2020-07-04 11:10:30,475 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a3693ac0>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:10:30,477 - meraki - DEBUG - {'tags': ['Networks'], 'operation': 'getOrganizationNetworks', 'method': 'GET', 'url': '/organizations/764320/networks', 'params': {}}
2020-07-04 11:10:30,479 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:10:30,611 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/organizations/764320/networks HTTP/1.1" 302 None
2020-07-04 11:10:30,614 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:10:31,086 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/organizations/764320/networks HTTP/1.1" 200 None
2020-07-04 11:10:31,088 - meraki - INFO - Networks, getOrganizationNetworks - 200 OK
2020-07-04 11:10:31,089 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:31] "[37mGET /networks HTTP/1.1[0m" 200 -
2020-07-04 11:10:32,374 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:32] "[37mPOST /clients HTTP/1.1[0m" 200 -
2020-07-04 11:10:32,378 - meraki - INFO - Meraki dashboard API session initialized with these parameters: {'self': <meraki.rest_session.RestSession object at 0x7f06a363dee0>, 'logger': <Logger meraki (DEBUG)>, 'api_key': '************************************e774', 'base_url': 'https://api.meraki.com/api/v0', 'single_request_timeout': 60, 'certificate_path': '', 'requests_proxy': '', 'wait_on_rate_limit': True, 'nginx_429_retry_wait_time': 60, 'action_batch_retry_wait_time': 60, 'retry_4xx_error': False, 'retry_4xx_error_wait_time': 60, 'maximum_retries': 2, 'simulate': False, 'be_geo_id': None, 'caller': None, '__class__': <class 'meraki.rest_session.RestSession'>}
2020-07-04 11:10:32,380 - meraki - DEBUG - {'tags': ['Clients'], 'operation': 'getNetworkClients', 'page': 1}
2020-07-04 11:10:32,383 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.meraki.com:443
2020-07-04 11:10:32,510 - urllib3.connectionpool - DEBUG - https://api.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/clients?perPage=1000&timespan=3600 HTTP/1.1" 302 None
2020-07-04 11:10:32,515 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): n210.meraki.com:443
2020-07-04 11:10:33,029 - urllib3.connectionpool - DEBUG - https://n210.meraki.com:443 "GET /api/v0/networks/L_681169443639788368/clients?perPage=1000&timespan=3600&perPage=1000&timespan=3600 HTTP/1.1" 200 None
2020-07-04 11:10:33,031 - meraki - INFO - Clients, getNetworkClients; page 1 - 200 OK
2020-07-04 11:10:33,032 - werkzeug - INFO - 127.0.0.1 - - [04/Jul/2020 11:10:33] "[37mGET /clients HTTP/1.1[0m" 200 -
